src/HealthChecks.AzureServiceBus/AzureServiceBusQueueHealthCheck.cs

```diff
@@ -33,8 +33,9 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
         {
             try
             {
-                var managementClient = ManagementClientConnections.GetOrAdd(ConnectionKey, _ => CreateManagementClient());
-                _ = await managementClient.GetQueueRuntimePropertiesAsync(_queueName, cancellationToken);
+                await using var client = ClientConnections.GetOrAdd(ConnectionKey, _ => CreateClient());

```

Comment:
`using` here 😕 ? Clients from dictionary are long-lived

Reply:
Sorry about that. I think making the AzureServiceBusHealthCheck class implement IAsyncDisposable is the right move but not 100%. Let me know if that looks good.

Reply:
Totally just realized I was dealing with a static property so I'm guessing they should actually just never be disposed.

Reply:
Yes